Tella/Scenes/Nextcloud/ViewModels/NextcloudReportViewModel.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  NextcloudReportViewModel.swift
+//  Tella
+//
+//  Created by Dhekra Rouatbi on 2/7/2024.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class NextcloudReportViewModel: ReportMainViewModel {
+    
+    override func getReports() {
+        
+        let draftReports =  tellaData?.database.getDriveReports(reportStatus: [.draft]) ?? []
+        let outboxedReports = tellaData?.database.getDriveReports(reportStatus: [.draft]) ?? []
+        let submittedReports = tellaData?.database.getDriveReports(reportStatus: [.draft]) ?? []
+        
+        draftReportsViewModel = draftReports.compactMap { report in
+            ReportCardViewModel(report: report,
+                                serverName: report.server?.name,
+                                deleteReport: {self.deleteReport(report: report)})
+        }
+        
+        outboxedReportsViewModel = outboxedReports.compactMap { report in
+            ReportCardViewModel(report: report,
+                                serverName: report.server?.name,
+                                deleteReport: {self.deleteReport(report: report)})
+        }
+        
+        submittedReportsViewModel = submittedReports.compactMap { report in
+            ReportCardViewModel(report: report,
+                                serverName: report.server?.name,
+                                deleteReport: {self.deleteReport(report: report)})
+        }
+    }
+    
+    override func listenToUpdates() {
+        self.mainAppModel.tellaData?.shouldReloadGDriveReports
+            .receive(on: DispatchQueue.main)
+            .sink { result in
+            } receiveValue: { draftReports in
+                self.getReports()
+            }.store(in: &subscribers)
+    }
+    
+    func deleteReport(report:GDriveReport) {
+        guard let reportId = report.id else { return }
+        //        let resultDeletion = self.tellaData?.deleteGDriveReport(reportId: reportId)
+        var message = ""
+        //        if case .success = resultDeletion {
+        //            message = String.init(format: LocalizableUwazi.uwaziDeletedToast.localized, entity.title ?? "")
+        //        } else {
+        //            message = LocalizableCommon.commonError.localized
+        //        }
+        
+        self.shouldShowToast = true
```

Comment:
do we need this code?

Reply:
hmm we don't need any of the nextcloud for this iteration. This was just for testing purposes of the new report flow. @dhekra-rouatbi are you ok with removing the nexcloud for this? I think you already have a different version in the branch you are working on right? 

We can also leave it as it is because it is going to be changed with the nextcloud release